docs/boards/nrf52840_dongle.md

```diff
@@ -59,14 +59,50 @@ Follow these steps:
 
     ![Nordic dongle retainer clip](../img/dongle_clip.jpg)
 
-1.  Depending on the programmer you're using, you may have to adapt the next
-    command line. Run our script for compiling/flashing Tock OS on your device:
+#### JLink
 
-    ```shell
-    $ ./deploy.py --board=nrf52840_dongle --programmer=jlink
+Depending on the programmer you're using, you may have to adapt the next
+command line. Run our script for compiling/flashing Tock OS on your device:
+
+```shell
+$ ./deploy.py --board=nrf52840_dongle --programmer=jlink
+```
+
+#### OpenOCD
+
+1. Create your openocd config, named `nordic_nrf52840_dongle.cfg` for st-link that 
+   might be:
+    ```
+    # Specific setup
+    source [find interface/stlink-dap.cfg]
+    transport select dapdirect_swd
+    
+    # The rest should be kept the same
+    set CHIPNAME nrf52840
+    source [find target/nrf52.cfg]
+    ```
+    
+    This file *must* reside inside a directory named `board`, for example:
     ```
+    .local/share/openocd
+    └── scripts 
+       └── board
+          └── nordic_nrf52840_dongle.cfg
+    ```
+1. Test your config - pass the *absolute* path using the `-s` flag, i.e.:
+       
+    ```
+    openocd -s /home/foo/.local/share/openocd/scripts/ -f board/nordic_nrf52840_dongle.cfg
```

Comment:
Have you tried to put the file in `${HOME}/.openocd/board` instead? This should be part of the default search locations, unlike `${HOME}/.local`

Reply:
That does seem to work. Weirdly, putting the config file in `/usr/share/openocd/scripts/board/` didn't work either :thinking: 
I'll update that part accordingly.